hardware_interface/include/hardware_interface/resource_manager.hpp

```diff
@@ -186,6 +186,48 @@ class HARDWARE_INTERFACE_PUBLIC ResourceManager
    */
   void import_component(std::unique_ptr<SystemInterface> system);
 
+  /// Import a hardware component which is not listed in the URDF
+  /**
+   * Components which are initialized outside a URDF can be added post initialization.
```

Comment:
What do you think about adding one more line here? (Also for other `import_component` instances if you agree on it)

```suggestion
   * Components which are initialized outside a URDF can be added post initialization.
   * Nevertheless, there should still be `HardwareInfo` available for this component, either parsed from a URDF string (easiest) or filled manually.
```